trade-finance-manager-ui/templates/case/facility/_macros/facility_details.njk

```diff
@@ -110,6 +111,21 @@
         <dt class="govuk-grid-column-one-quarter govuk-body-s">Cover end date</dt>
         <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="facility-cover-end-date">{{ facility.dates.coverEndDate | dashIfEmpty }}</dd>
 
+        {% if isTfmFacilityEndDateFeatureFlagEnabled %}
```

Comment:
I think I've seen you've checked with Ahila but to confirm, if a existing version 0 deal has been submitted to UKEF, we want to show a `-` to these questions? rather than hide the section altogether

Reply:
Yep that's correct - if an existing version 0 deal is being viewed, while the feature flag is enabled, then it should show 'Bank has provided a facility end date' with a dash, and 'Facility end date' with a dash too.